### DISCUSSION TITLE
关于播放界面交互

### DISCUSSION THREAD

**Discussion by Echo7251** [Richasy/Bili.Uwp#758] [想法]
在播放界面，右键空白区域呼出下方播放组件，再右键却没法让它回去了。这时候需要暂停再播放来隐藏播放组件，没有实质性伤害，但很恼人。
既然右键能够呼出播放组件，希望再次右键能够隐藏组件。

**Comment by Richasy** [comment]
没有右键操作，左键即可

**Reply by Echo7251** [reply]
啊，我还以为右键呼出组件是特意设计的。不过左键的话会暂停 … 那我还是放弃双击全屏吧。
不过说回来应用真的很不错，请加油
